<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>8</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="userNameHorizontalLayout">
           <item>
            <widget class="QLabel" name="userNameLabel">
             <property name="text">
              <string>User name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userNameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="userAgeHorizontalLayout">
           <item>
            <widget class="QLabel" name="userAgeLabel">
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userAgeLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="startGamePushButton">
           <property name="text">
            <string>Start Game</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="vehicleIdentifierLabel">
             <property name="text">
              <string>Vehicle Identifier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="vehicleIdentifierLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="changeStatePushButton">
           <property name="text">
            <string>Change State</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
